src/Containers/EditControlPanel/EditControlPanel.tsx

```diff
@@ -0,0 +1,160 @@
+import React, {useState} from 'react';
+import styled from "styled-components";
+import Button from "../../Inputs/Button/Button"
+
+export interface IEditControlPanel {
+    /**
+     * The table number for the table
+     */
+    TableNumber: string;
+    /**
+     * The number of seats at the table
+     */
+    NumberOfSeats: number;
+}
+
+/**
+ * Primary UI component for user interaction
+ * Edit control panel for the right side of the edit page
+ */
+export const EditControlPanel: React.FC<IEditControlPanel> = ({
+    TableNumber= 'T1',
+    NumberOfSeats = 1,
+    ...props
+}) => {
+
+    const [seats, setSeat] = useState(NumberOfSeats);
+    return (
+        <BorderForControlPanel {...props}>
+            <CenteredText>
+                <FontStylesAndColor>Table Number</FontStylesAndColor>
+                <InputStyles placeholder={TableNumber} />
+            </CenteredText>
+            <CenteredText>
+                <FontStylesAndColor>Number Of Seats</FontStylesAndColor>
+                <Border>
+                    <Container>
+                        <Row>
+                            <Col1>
+                                <StylesForLeftButton onClick={() => setSeat(seats - 1)}>-</StylesForLeftButton>
```

Comment:
this won't work, use prevState += 1 look at the React Docuemntation (Counter Example)